packages/flutter/lib/src/rendering/viewport.dart

```diff
@@ -1040,6 +1055,10 @@ abstract class RenderViewportBase<ParentDataClass extends ContainerParentDataMix
   ///
   /// The `duration` parameter can be set to a non-zero value to animate the
   /// target object into the viewport with an animation defined by `curve`.
+  ///
+  /// See also:
+  ///
+  /// * [RenderObject.showOnScreen], commonly used to wrap `showInViewport`.
```

Comment:
It seems like only two classes(RenderViewportBase, SingleChildScrollView) are using this in their show on screen. Let's list two of them separately here